TimePicker/tests/TimePicker-specs.js

```diff
@@ -163,4 +195,44 @@ describe('TimePicker', () => {
 
 		expect(header).toBeNull();
 	});
+
+	test('\'timeToLocaleString\' method should return \'null\' for an \'undefined\' time', () => {
+		const time = timeToLocaleString(undefined);		// eslint-disable-line no-undefined
+
+		expect(time).toBeNull();
+	});
+
+	test('should change the meridiem to reflect the hour change', () => {
+		const time = new Date(2000, 0, 1, 12, 30);
+		const secondTime = new Date(2000, 0, 1, 11, 30);
+		const {rerender} = render(
+			<TimePicker locale="en-US" value={time} />
+		);
+		const firstTimeDisplayed = screen.queryByText('12:30 PM');
+
+		rerender(
+			<TimePicker locale="en-US" value={secondTime} />
+		);
+		const secondTimeDisplayed = screen.queryByText('11:30 AM');
+
+		expect(firstTimeDisplayed).not.toBeNull();
+		expect(secondTimeDisplayed).not.toBeNull();
+	});
+
+	test('check that the date is displayed correctly for cases of more than 2 meridiems', () => {
+		const time = new Date(2000, 0, 1, 12, 30);
+		const secondTime = new Date(2000, 0, 1, 11, 30);
+		const {rerender} = render(
+			<TimePicker locale="am-ET" value={time} />
+		);
+		const firstTimeDisplayed = screen.queryByText('12:30 PM');
```

Comment:
I think queryByText('12:30 PM' ) is not proper in this case as this TimePicker is set to locale="am-ET" not locale="en-US". Can you please check it?

Reply:
Done